src/systems/capture-system.js

```diff
@@ -0,0 +1,64 @@
+export class CaptureSystem {
+  constructor(scene) {
+    this.scene = scene;
+
+    this.gotAudioTrack = false;
+    this.stream = new MediaStream();
+
+    this.scene.addEventListener("loaded", () => {
```

Comment:
I think it may be possible for the scene to have already been loaded. I know that is surprising but humor me for a moment.

The `a-scene` element's prototype (aka `AScene.prototype`) is a an `AEntity.prototype`, which in turn is an `ANode.prototype`. The `ANode` is responsible for emitting the `loaded` event, which happens once it notices that all of its children have been `loaded`. This typically happens in an `AEntity`'s `attachedCallback`, but doesn't in the case that the `AEntity` `isScene`. In the case of an `AScene`, `load` is called from a `setTimeout` in `play`, which is called from `attachedCallbackPostCamera`, which happens when the `cameraready` event handler fires....

This happens in the `camera` system's `init` which is.... 
Ok it's after the `hubs-systems` `init`, so nevermind, I think you're fine.

If the camera system came before... then... would it break? I thiinknk not... because the `timeout` in `AScene`'s `play` would make all the system `init`s get called before its `load`. But if that timeout wasn't there, then could it be?

OK feel free to ignore this. I was going to suggest you check if the scene `hasLoaded` but I think that isn't required after all.